GitHub/GitHub.Octokit.csproj

```diff
@@ -1,10 +1,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <PackageId>Octokit.net</PackageId>

```

Comment:
This needs to be dotnet-sdk or something other than Octokit.net - that package already exists.

Reply:
Does it? The original Octokit.net uses ["Octokit"](https://github.com/octokit/octokit.net/blob/main/Octokit/Octokit.csproj#L11) as its PackageId. 

I'm open to suggestions here! I considered `Octokit.NET.SDK` and `Octokit.NET.Generative` as well.

Reply:
Oh you are correct, it resolves on nuget as `octokit`.  Yeah I think we should go with something different from any existing moniker we use.

I could get behind `Octokit.NET.SDK` or perhaps `GitHub.NET.SDK`?

Reply:
I've changed it now to Octokit.NET.SDK!